src/config.c

```diff
@@ -24,6 +24,33 @@
 #include <ctype.h>
 #include <sys/stat.h>
 
+
+const struct vpn_config invalid_cfg = {
+	.gateway_host = {'\0'},
+	.gateway_port = 0,
+	.username = {'\0'},
+	.password = {'\0'},
+	.otp = {'\0'},
```

Comment:
Writing it there to not forget, I noticed `--otp` isn't in the man page, but it is in `--help`.